pyamg/tests/test_multilevel.py

```diff
@@ -167,3 +167,10 @@ def test_coarse_grid_solver(self):
                 # subsequent calls use cached data
                 x = s(A, b)
                 assert_almost_equal(A*x, b)
+
+
+class TestChangeMatrix(TestCase):
+    """Test changing the fine level matrix."""
+
+    def test_change_matrix(self):
+        pass
```

Comment:
Are we skipping a unit test for te change matrix funciton? I know you had tested the lin/quad change

Reply:
good catch. I guess I put it in `test_smoothing.py` but I could add the short lin/quad case here...